installer/templates/new/assets/javascripts/widget.js

```diff
@@ -23,6 +23,17 @@ class Widget extends React.Component {
           setTimeout((() => window.location.reload()), 5 * 60 * 1000)
         }
       });
+
+      this._events.addEventListener('_kitto', (event) => {
```

Comment:
Btw i already tried the above and works with your branch. I can push you a fixup if you want.